meilisearch-schema/src/schema.rs

```diff
@@ -190,11 +190,30 @@ impl Schema {
         Ok(())
     }
 
-    pub fn index_new_fields(&self) -> bool {
-        self.index_new_fields
+    pub fn set_all_fields_indexed(&mut self) {
+        self.indexed.clear();
+        self.indexed_map.clear();
+
+        for (_name, id) in self.fields_map.iter() {
+            let pos = self.indexed.len() as u16;
+            self.indexed.push(*id);
+            self.indexed_map.insert(*id, pos.into());
+        }
+    }
+
+    pub fn set_all_fields_displayed(&mut self) {
```

Comment:
@Kerollmops, what do you think about this function name?

Reply:
I would prefer something like `set_all_fields_as_displayed`.